docs/index.adoc

```diff
@@ -150,31 +150,53 @@ It *must be unique* for your application.
 
 NOTE: See <<Feature Permissions>> for details on the various states a feature can have.
 
-=== Split targeting strategies and targeting rules
+=== Strategy Splits and general targeting rules
 
-Split targeting provides an ability to rollout features to a limited audience based on targeting rules, for example if you want to turn your feature on 
-for a user only on a mobile device or only with a specific email or userId. Apart from the targeting rules you can also split by percentage, or mix attributes with percentages.
+We refer to a feature flag in an as a Rollout Strategy. Each Rollout Strategy can have _multiple_
```

Comment:
Not sure why you removed my description. I think we should have it as well as what you have added. Mine just gives a high level example and yours goes into details of having multiple strategies.